src/components/ActivitiesList/ActivitiesDataFetch.tsx

```diff
@@ -0,0 +1,52 @@
+import { Activity } from "@components/ActivitiesList/ActivitiesList";
+import axios from "axios";
+import React, { ReactElement, useEffect, useState } from "react";
+
+import styles from "./ActivitiesList.module.css";
+
+const URL = "http://localhost:8000/api/v1/activities";
+
+const ActivitiesList: React.FC = (): ReactElement => {
+  const [activities, setActivities] = useState<Activity[]>([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const {
+          data: { activities },
+        } = await axios.get(URL);
+        setActivities(activities);
+      } catch (error) {
+        console.error("Error fetching activities:", error);
+        if (axios.isAxiosError(error)) {
+          return {
+            data: "API request failed! Did you remember to `npm run dev` the backend app?",
+          };
+        }
+        return {
+          data: "A fetching data error occurred.",
+        };
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  return (
+    <div className={styles.activitiesContainer}>
+      <h1>Activities</h1>
+      <div className={styles.activitiesList}>
+        {activities.map((activity) => (
+          <div key={activity._id} className={styles.activityItem}>
+            <p>Activity Name: {activity.name}</p>
+            <p>Description: {activity.description}</p>
```

Comment:
Since you don't need to ever do anything with the `activity` object as a whole, try destyructuring its data in the parameter definition, so you use less of the dot notation:

```suggestion
        {activities.map({_id, name, description, votes} => (
          <div key={activity._id} className={styles.activityItem}>
            <p>Activity Name: {activity.name}</p>
            <p>Description: {activity.description}</p>
```

if you needed to pass the entire `activity` to a subcomponent etc, you could leave it as `activity` in the parameter, but then do the destructuring on the first line of the function body. 

Sometimes when things are complicated and a lot of data is flying around it's more readable to see a lot of dot notation, but in this case it's pretty straightforward so you can save some work